AirCasting/SearchAndFollow/SearchMap/SearchMapView.swift

```diff
@@ -60,16 +60,16 @@ struct SearchMapView: View {
             })
             .alert(item: $viewModel.alert, content: { $0.makeAlert() })
             .sheet(isPresented: $viewModel.isLocationPopupPresented, onDismiss: {
-                viewModel.redoTapped()
+                viewModel.dismissedTapped()
```

Comment:
I dont quite like this name. It suggests a bit that some *map* dismiss button was tapped. How about `locationPopupDisimssed()`